openlibrary/core/lending.py

```diff
@@ -56,7 +55,8 @@ def setup(config):
     global config_content_server, config_loanstatus_url, \
         config_ia_access_secret, config_bookreader_host, \
         config_ia_ol_shared_key, config_ia_ol_auth_key, \
-        config_ia_ol_xauth_s3, config_internal_api_key
+        config_ia_ol_xauth_s3, config_internal_api_key, \
+        IA_API_URL, AVAILABILITY_API, IA_XAUTH_API_URL
```

Comment:
It'd be nice to follow the convention of calling these constants `config_*` so that it's always clear their values come from the config.